src/ui-main.js

```diff
@@ -0,0 +1,181 @@
+/*package annotator.ui */
+"use strict";
+
+var adder = annotator.ui.adder;
+var editor = aca.ui.editor;
+var highlighter = annotator.ui.highlighter;
+var textselector = annotator.ui.textselector;
+var viewer = annotator.ui.viewer
+
+// Returns the absolute position of the mouse relative to the top-left rendered
+// corner of the page (taking into account padding/margin/border on the body
+// element as necessary).
+function mousePosition(event) {
+    var body = document.body;
+    var offset = {top: 0, left: 0};
+
+    if ($(body).css('position') !== "static") {
+        offset = $(body).offset();
+    }
+
+    return {
+        top: event.pageY - offset.top,
+        left: event.pageX - offset.left
+    };
```

Comment:
Checking if body's position is static seems hacky here... when would body's position change?